src/style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.bigForm {
+  position: relative;
+  margin: 20px 0;
+  font-weight: 300;
+}
+
+.input {
+  outline: none;
+  height: 70px;
+  font: inherit;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  width: 100%;
+  padding: 0 0 0 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 800;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.images {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.smallForm {
+  margin: 20px 0;
+}
+
+.input--small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 42px;
+  padding: 0 0 0 33px;
+}
+
+.input--small::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.images--small {
+  width: 11px;
+  height: 11px;
+  top: 125px;
```

Comment:
image should be connecter to its form, not `body`, add `position: relative` to all forms